internal/codegen/golang/templates/pgx/queryCode.tmpl

```diff
@@ -103,6 +103,23 @@ func (q *Queries) {{.MethodName}}(ctx context.Context, {{.Arg.Pair}}) (int64, er
 }
 {{end}}
 
+{{if eq .Cmd ":execlastinsertid"}}
+{{range .Comments}}//{{.}}
+{{end -}}
+{{if $.EmitMethodsWithDBArgument -}}
+func (q *Queries) {{.MethodName}}(ctx context.Context, db DBTX, {{.Arg.Pair}}) (int64, error) {
+	result, err := db.Exec(ctx, {{.ConstantName}}, {{.Arg.Params}})
+{{- else -}}
+func (q *Queries) {{.MethodName}}(ctx context.Context, {{.Arg.Pair}}) (int64, error) {
+	result, err := q.db.Exec(ctx, {{.ConstantName}}, {{.Arg.Params}})
+{{- end}}
+	if err != nil {
+		return 0, err
+	}
+	return result.LastInsertId(), nil
```

Comment:
pgx doesn't support driver.Result.LastInsertId. People usually do "INSERT INTO ... RETURNING id" with :one when they want the sequence number from Postgres.

You should probably return an error when people try to use :execinsertid with Postgres.

Reply:
pgx also doesn't support the same signature for `RowsAffected()`. We'll need to address this is a different PR.